docs/Q-and-A.md

```diff
@@ -0,0 +1,635 @@
+
+#### **Q: Why only a protocol, why scalable, why decentralized, why a ledger, why permissionless, why open and public?
+1. Sidetree is a _protocol_ because we need it to be multi applicable. Implementers will be provided with software tools and toolkits and the protocol has both `normative` and non-normative guidelines.
+2. The _scalability_ feature is needed because of the astronomical numbers that are associated with digital identifiers, the numerous life events of each identifier and the mulitple use of each individual identifier. Every individual, group or thing on earth can have multiple identifiers associated with them.
+3. Privacy and guarding control over are the main drivers for the deer wish to _decentralize_ identifiers. If you don’t like being tracked and you don’t like anybody else controlling your identity, decentralization is the way to go. 
+4. A global _ledger_ is needed as a single source of truth.
+5. Permissioned identifiers are limited to a certain domain of application. Sidetree does not want to confine itself and will be as _permissionless_ as the underlaying ledger.
+6. Lastly Sidetree needs to be open and public and at the same the protocol and its applications need to hide and protect all sensitive identity related data. This seems to contradict each other. However in Sidetree it is doable. Cryptography and the protocol safeguard privacy and the _open public_ characteristic offers availability and inclusion at the same time.
+(_@henkvancann_)
+
+## **Q: What is the setting of sidetree?
+The protocol allows users to create _globally unique, user-controlled identifiers_ and manage their associated metadata, all _without the need for centralized authorities_ or trusted third parties. The syntax of the identifier and accompanying data model used by the protocol is conformant to that defined by W3C Decentralized Identifiers. Implementations of the protocol can be codified as their own distinct DID Methods and registered in the W3C DID Method Registry.
+
+## *Q: What is in the name 'Sidetree'?
+The name “Sidetree” combines two hints as to its technical inspirations and superpowers. Each Sidetree network functions as a kind of identity-specific “sidechain” periodically pinned to an underlying chain in much the same way that so-called “Layer 2 Networks”.  These include the Lightning network running atop Bitcoin.\
+(_@bumblefudge_)
+
+## **Q: How is the inner working of Sidetree?
+<img src="../www/diagrams/architecture.png" alt="Architecture of Sidetree" border="0" width="400">
+
+#### ***Q: Why have the concept of index files?
+It would be ideal to be able to fetch metadata about the batched operations efficiently, without needing to download the entire batched operation. This design is needed for the implementation of "light nodes".
+
+## *Q: What are the main components of a sidetree system?
+The figure below illustrates the three primary components of a Sidetree-based DID system:
+1. Single source of truth
```

Comment:
"Single source" is more the language of a public blockchain implementation.

Reply:
Changed into : 1. Single point of definition